crimsobot/utils/tools.py

```diff
@@ -68,10 +68,12 @@ def crimbed(title: Optional[str], descr: Optional[str], thumb_name: Optional[str
         'triumph': 'https://i.imgur.com/bBXRFnO.png',
         'joy': 'https://i.imgur.com/8deo8Ak.png',
         'hug': 'https://i.imgur.com/lSPKbWf.png',
-        'think': 'https://i.imgur.com/odD9yI2.png',
+        # 'think': 'https://i.imgur.com/odD9yI2.png',
```

Comment:
nit: remove commented out code

Reply:
i am going to keep that old link *for now*

Reply:
but it's in the git history! you don't need to leave it! :D